tfhe/src/high_level_api/integers/types/base.rs

```diff
@@ -1678,3 +1678,145 @@ where
         GenericInteger::<P>::new(ciphertext, self.id)
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use crate::conformance::ParameterSetConformant;
+    use crate::core_crypto::prelude::UnsignedInteger;
+    use crate::shortint::parameters::PARAM_MESSAGE_2_CARRY_2_KS_PBS;
+    use crate::shortint::{CiphertextModulus, PBSOrder};
+    use crate::{generate_keys, set_server_key, ConfigBuilder, FheUint8};
+    use rand::{thread_rng, Rng};
+
+    type IndexedParameterAccessor<Ct, T> = dyn Fn(usize, &mut Ct) -> &mut T;
+
+    type IndexedParameterModifier<'a, Ct> = dyn Fn(usize, &mut Ct) + 'a;
+
+    fn change_parameters<Ct, T: UnsignedInteger>(
+        func: &IndexedParameterAccessor<Ct, T>,
+    ) -> Vec<Box<IndexedParameterModifier<'_, Ct>>> {
```

Comment:
Very minor thing, change_parameters could return an array instead of a vec

Reply:
Done